src/lib/firebase.js

```diff
@@ -12,6 +12,7 @@ const firebaseConfig = {
   storageBucket: 'tcl-2-smart-shopping-list.appspot.com',
   messagingSenderId: '538706887514',
   appId: '1:538706887514:web:30b4235c1e8aa5fa19a963',
+  timestampsinSnapshots: true
```

Comment:
Were we getting errors? I _thought_ Firebase no longer required this property, but I could be wrong. Would be good to know the current truth around this so we can update `firebase.js` for the next cohort. Thanks!

Reply:
I actually entirely forgot we added this. I think we were thinking "Oh! Time! There was a note about that back when we were starting with the database" so we added it in thinking we might need it :) But we didn't - I'll remove it!

Reply:
OK, cool!